### PR TITLE
[csgo] add CSGO stats

### PR DESCRIPTION
**Pull request by obviyus**

### CODE REVIEW

File: configuration/__init__.py [hunk 1]
```diff
@@ -6,19 +6,20 @@
...
-        "FOR_WHAT_ID": os.environ["FOR_WHAT_ID"],
+        "MYSQL_IP_ALIAS": os.environ["MYSQL_IP_ALIAS"],
+        "MYSQL_PASSWORD": os.environ["MYSQL_PASSWORD"],
+        "MYSQL_USER": os.environ["MYSQL_USER"],
+        "OPENWEATHER_API_KEY": os.environ["OPENWEATHER_API_KEY"],
         "PUNYA_SONG_ID": os.environ["PUNYA_SONG_ID"],
-        "AUDIO_RESTORE_USERS": os.environ["AUDIO_RESTORE_USERS"].split(),
+        "STEAM_API_KEY": os.environ["STEAM_API_KEY"],
+        "TELEGRAM_BOT_TOKEN": os.environ["TELEGRAM_BOT_TOKEN"],
+        "WOLFRAM_APP_ID": os.environ["WOLFRAM_APP_ID"],
```
Comment (Tyagdit) [inline configuration/__init__.py]: bro these were grouped according to uses, pls restore. Also remove mysql stuff

File: api/csgo.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+STEAM_API_KEY = config["STEAM_API_KEY"]
+
+
+def make_csgo_response(username: str, steam_id: str) -> str:
+    url: str = "https://api.steampowered.com/ISteamUserStats/GetUserStatsForGame/v0002/"
+    params: Dict[str, str] = {"appid": "730", "key": STEAM_API_KEY, "steamid": steam_id}
+
+    try:
+        response: Dict[str, Dict] = requests.get(url, params).json()
+        username: str = username.replace('_', '\_')
```
Comment (Tyagdit) [inline api/csgo.py:31]: this is a bad idea, try Markdown_v2 parsemode

File: api/csgo.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+
+def set_steam_id(update: 'telegram.Update', context: 'telegram.ext.CallbackContext') -> None:
+    """Set your SteamID"""
+    if update.message:
+        message: 'telegram.Message' = update.message
+    else:
+        return
+
+    username: str = ' '.join(context.args).strip() if context.args else ''
```
Comment (Tyagdit) [inline api/csgo.py]: `strip()` is redundant

**Review by Tyagdit — CHANGES_REQUESTED** [review]

File: api/csgo.py (hunk 2 above)
Comment (obviyus) [reply api/csgo.py:31]: Let's make this a separate PR.

File: Dockerfile [hunk 4]
```diff
@@ -12,10 +12,10 @@ COPY / .
 
 # install dependencies
 RUN apt-get update \
-    && apt-get upgrade -y \
```
Comment (Tyagdit) [inline Dockerfile:15]: why remove upgrade?

File: api/csgo.py [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    else:
+        return
+
+    username: str = ' '.join(context.args) if context.args else ''
+    text: str
+
+    if not username:
+        text = "*Usage:* `/setid {STEAM_ACCOUNT_NAME}`\n" \
+               "*Example:* `/setid obviyus`\n" \
+               "This is not your steam username, but your steam account name."
```
Comment (Tyagdit) [inline api/csgo.py]:
```suggestion
               "This is not your steam username, but your steam account name or steam ID."
```

File: api/csgo.py [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+    elif ' ' in username:
+        text = "This command does not work with Steam usernames."
+    elif len(username) == 17 and username.isdigit():
+        try:
+            url: str = "https://api.steampowered.com/ISteamUserStats/GetUserStatsForGame/v0002/"
+            params: Dict[str, str] = {"appid": "730", "key": STEAM_API_KEY, "steamid": username}
+            response: Dict[str, Dict] = requests.get(url, params).json()
+
+            _ = response["playerstats"]["stats"]
```
Comment (Tyagdit) [inline api/csgo.py:99]: add a comment explaining this jugaad

File: Dockerfile (hunk 4 above)
Comment (obviyus) [reply Dockerfile:15]: https://github.com/hadolint/hadolint/wiki/DL3005

**Review by Tyagdit — APPROVED** [review]

**Pull request merged**